lib/reporter.js

```diff
@@ -57,7 +49,7 @@ var CoverageReporter = function (rootConfig, helper, logger, emitter) {
 
   function normalize (key) {
     // Exclude keys will always be relative, but covObj keys can be absolute or relative
-    var excludeKey = isAbsolute(key) ? path.relative(basePath, key) : key
+    var excludeKey = path.isAbsolute(key) ? path.relative(basePath, key) : key
```

Comment:
AFAICT this commit is not related to major libs deps. Please move to separate Pull Request.

Reply:
Done